ipn/ipnlocal/local.go

```diff
@@ -516,6 +518,11 @@ func (b *LocalBackend) Shutdown() {
 		b.sshServer = nil
 	}
 	b.closePeerAPIListenersLocked()
+	if b.debugSink != nil {
+		b.e.InstallCaptureHook(nil)
```

Comment:
are you missing a call to InstallCaptureHook(nil) when the Sink's number of Outputs transitions from 1 to 0? I'm not seeing it.

Reply:
The sink and the hook stays installed until the `LocalBackend` shuts down (the logic to close the sink is in Close). Are you saying you want to attempt teardown when there are no outputs?

Seems doable but might be a lil messy trying to manage concurrency with other `StreamDebugCapture` calls

Reply:
I've gone ahead and made it shut down the sink if there are no more outputs.